e2e/node/e2e.test.ts

```diff
@@ -148,37 +148,92 @@ describe("End-to-end verifiable credentials tests for environment", () => {
 
   describe("lookup VCs", () => {
     it("returns all VC issued matching a given shape", async () => {
-      const result = await getVerifiableCredentialAllFromShape(
-        new URL("derive", env.vcProvider).href,
-        {
-          "@context": [
-            "https://www.w3.org/2018/credentials/v1",
-            "https://schema.inrupt.com/credentials/v1.jsonld",
-          ],
-          type: ["VerifiableCredential", "SolidAccessGrant"],
-          credentialSubject: {
-            id: vcSubject,
-            providedConsent: {
-              hasStatus:
-                "https://w3id.org/GConsent#ConsentStatusExplicitlyGiven",
+      const [credential1, credential2] = await Promise.all([
+        issueVerifiableCredential(
+          new URL("issue", env.vcProvider).href,
+          validSubjectClaims({ resource: "https://example.org/some-resource" }),
+          validCredentialClaims,
+          {
+            fetch: session.fetch,
+          },
+        ),
+        issueVerifiableCredential(
+          new URL("issue", env.vcProvider).href,
+          validSubjectClaims({
+            resource: "https://example.org/another-resource",
```

Comment:
Why not make the resource something in the owners storage?

Reply:
These tests are testing the VC layer of things, so we're not going to exercise the Access Grant that gets issued because that would be going beyond just the VC part of it. We happen to issue an Access Grant merely because the ESS issuer is configured to only issue a restricted set of VCs, but ideally these would just issue arbitrary VCs. From there, the data in the grant is arbitrary, that's why I used an arbitrary IRI and not an actual resource that would need to be created and then deleted, adding some surface to the test that I don't think brings value.